### DISCUSSION TITLE
psr/simple-cache

### DISCUSSION THREAD

**Discussion by urshar** [PHPOffice/PhpSpreadsheet#2575] [General]
Hi,

I'm starting a new Laravel project (Laravel 9) and need maatwebsite/excel for it. However, I can't install this app because phpSpreadsheet specifies version 1.0 of psr/simple-cache. Is there a special reason for this or will newer versions of psr/simple-cache also be supported in the next version?

Andreas

**Comment by MarkBaker** [comment]
Have a read of [Issue #2468](https://github.com/PHPOffice/PhpSpreadsheet/issues/2468)

**Comment by oleibman** [comment]
composer.json has for some time specified
```
"psr/simple-cache": "^1.0 || ^2.0 || ^3.0"
```
